frontend/src/Components/CardSelector.tsx

```diff
@@ -4,25 +4,31 @@ import classes from './CardSelector.module.css';
 import { IconCoffee } from './IconCoffee';
 import { connectToWebSocket } from './WebSocket';
 import { IconObserver } from './IconObserver';
-import { BUTTON_OBSERVER, VOTE_COFFEE, VOTE_OBSERVER } from '../constants';
+import { BUTTON_OBSERVER, VOTE_COFFEE, VOTE_NOTE_VOTED, VOTE_OBSERVER } from '../constants';
 
 const ProtoCardSelector = ({ socket }: { socket: WebSocketApi }) => {
   const selectedCard = socket.state.votes[socket.loginData.user];
+
   return (
     <>
       <div class={classes.cardCollection}>
-        {socket.state.scale.map((cardValue) => (
-          <button
-            key={cardValue}
-            class={classNames([
-              classes.largeCard,
-              { [classes.selected]: selectedCard === cardValue },
-            ])}
-            onClick={() => socket.setVote(cardValue)}
-          >
-            {cardValue === VOTE_COFFEE ? <IconCoffee /> : cardValue}
-          </button>
-        ))}
+        {socket.state.scale.map((cardValue) => {
+          return (
+            <button
+              key={cardValue}
+              class={selectedCard === cardValue ? classNames([classes.largeCard, classes.selected]) : classNames([classes.largeCard])}
```

Comment:
If you think this version is easier to understand, you should also align the observer button.

Reply:
I changed this because of poor understanding of how `classNames` work. I like you suggestion where you say:

> Last, thinking a little more about classNames, maybe it is easier instead of the mixed object form to make use of the fact that classNames prunes undefined values and write it like this:

And I will go with it